{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -378,6 +380,8 @@ def DATABASES(self):  # pragma: no cover
     # Security
     # https://docs.djangoproject.com/en/stable/topics/security/
 
+    CSRF_COOKIE_SAMESITE = "Strict"
```

Comment:
As mentioned above, also this seems like a customization for a specific project, it doesn't seem necessary to me include it in the bootstrap.